tuning/config/configs.py

```diff
@@ -0,0 +1,42 @@
+from dataclasses import dataclass, field
+from typing import Dict, Optional
+import transformers
+
+DEFAULT_CONTEXT_LENGTH=4096
+DEFAULT_OPTIMIZER="adamw_torch"
+
+IGNORE_INDEX = -100
```

Comment:
appears to be unused

Reply:
to fix the above comment, I have added the `DataCollator` constructor to take the `ignore_index` parameter and using the default one now. We have to change this based on the model.

@lchu-ibm -- another potential model specific code.